auth_provider/templates/auth/login.html

```diff
@@ -1,23 +1,108 @@
-{% extends "base.html" %} {% block content %} {% if form.errors %}
-<div class="alert">
-  <p>Username and password did not match</p>
-</div>
-{% endif %}
+{% extends "base.html" %} {% block content %} 
+
+<style>
+  body {
+    font-family: Arial, Helvetica, sans-serif;
+  }
+
+  * {
+    box-sizing: border-box;
+  }
+
+  /* Add padding to containers */
+  .container {
```

Comment:
I think we might also want 
```css
  max-width: 80ch;
  margin: 10vh auto 0 auto;
```
so that the page looks like
<img width="1762" alt="image" src="https://user-images.githubusercontent.com/43416725/155836382-28b54cf6-aebb-49de-9cb1-65bff6e09669.png">
instead of
<img width="1753" alt="image" src="https://user-images.githubusercontent.com/43416725/155836387-247ca900-db34-4af4-ad9e-71871351130e.png">

Reply:
Will put this change in!